libs/ml/FFICallbacks.ml

```diff
@@ -0,0 +1,7 @@
+type callbacks = int
+external ocaml_send_tcp2: callbacks -> string -> int = "ocaml_send_tcp"
+external ocaml_recv_tcp2: callbacks -> string -> int = "ocaml_recv_tcp"
+
+let ocaml_send_tcp callbacks b = ocaml_send_tcp2 callbacks b
+let ocaml_recv_tcp callbacks b = ocaml_recv_tcp2 callbacks b
+
```

Comment:
alright, starting this review

why do you need this indirection? I would assume you could get away with just an `mli` and:

```
type callbacks = int
external ocaml_send_tcp: callbacks -> string -> int = "c_ocaml_send_tcp"
external ocaml_recv_tcp: callbacks -> string -> int = "c_ocaml_recv_tcp"
```